official/mnist/mnist.py

```diff
@@ -125,7 +125,10 @@ def model_fn(features, labels, mode, params):
 
     logits = model(image, training=True)
     loss = tf.losses.sparse_softmax_cross_entropy(labels=labels, logits=logits)
-    accuracy = tf.metrics.accuracy(
+    if params.get('multi_gpu'):
+      accuracy = (tf.no_op(), tf.constant(0))
```

Comment:
This should not be necessary. If you are having issues make sure your version of tf-nightly is up to date.

Reply:
I tried to change from mirrored strategy to distribution_utils.get_distribution_strategy. While mirroed strategy needs just num_gpus, distribution_utils.get_distribution_strategy might need all_reduce_algo, inter_op_parallelism_threads and intra_op_parallelism_threads. I am not sure how to set and pass such parameters. Please let me know about them.